### ISSUE TITLE
Progress bar not correct with second print

### ISSUE THREAD

**Issue by Ranney1** [CR6Community/CR-6-touchscreen#22]
With the latest extui-dev release, the print progress bar is working correct.

When starting a second print (without turning the printer off between them) the progress bar will start at 100%.

Latest Marlin firmware for BTT mainboard and stock screen in use.

**Comment by Sebazzz** [comment]
It starts at 100%, but it does work after, doesn't it?

**Comment by Ranney1** [comment]
Yes, I see it now one hour after starting the print that it is back at 1%.

Ant thanks for the better translation ;-)
